### ISSUE TITLE
Purity and ploidy estimates from EM solution

### ISSUE THREAD

**Issue by kpjonsson** [mskcc/facets#27]
As far as I can see these estimates are not calculated anywhere. Could they be?

**Comment by shenmskcc** [comment]
Not sure what you mean. Tumor ploidy is computed from dipLogR. Tumor purity
is inferred from the cellular fraction estimates of the clonal copy number
events.

**Comment by kpjonsson** [comment]
But the ploidy estimate will differ if the naïve and EM solutions for the integer copy-number differs, no?

**Comment by shenmskcc** [comment]
no. ploidy is determined first using segments in balanced states before
integer copy number calls.
